docs/spec.md

```diff
@@ -1641,15 +1651,15 @@ Afterwards, `result@process` is given by:
 
 | Label | Name                  | Type                                         | Constraints |
 |-------|-----------------------|----------------------------------------------|-------------|
-| (I1)  | `operand`             | tensor                                       | (C5)        |
+| (I1)  | `operand`             | tensor or per-tensor quantized tensor        | (C5)        |
```

Comment:
Does `constant(0, element_type(result))` work correctly for the quantized case?

Reply:
What I meant is whether we should perhaps use `zero_point(result)` instead of `0` for the quantized case?

Reply:
I see your point. Thanks! 
I think that is the right choice as value 0 is not guaranteed to be represented in the the quantized value domain.

Reply:
Made similar changes for `imag` op.

Reply:
Let's revert to `constant(0, element_type(result))` and resolve as part of #1687. I think that there are multiple design options that need to be evaluated carefully, and we don't have to block this PR on that.

Reply:
Done.
Sure.